app/modules/ads1115.c

```diff
@@ -252,9 +252,6 @@ static int ads1115_lua_register(lua_State *L, uint8_t chip_id) {
         return luaL_error(L, "unexpected config value (%p) please reset device before calling this function", config_read);
     }
     ads_ctrl_ud_t *ads_ctrl = (ads_ctrl_ud_t *)lua_newuserdata(L, sizeof(ads_ctrl_ud_t));
-    if (NULL == ads_ctrl) {
-        return luaL_error(L, "ads1115 malloc: out of memory");
-    }
```

Comment:
Unrelated change?  (Though I think one we should take, but not part of this PR.)

Reply:
Ok. Let's make it another PR.